spec/request/email_request_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'email', type: :request do
+  include SchemaMatchers
+
+  let(:token) { 'fa0f28d6-224a-4015-a3b0-81e77de269f2' }
+  let(:auth_header) { { 'Authorization' => "Token token=#{token}" } }
+  let(:user) { build(:user, :loa3) }
+
+  before do
+    Session.create(uuid: user.uuid, token: token)
+    User.create(user)
+  end
+
+  describe 'GET /v0/profile/email' do
```

Comment:
please add specs for the non 200 http status codes EVSS lists in their swagger docs

Reply:
@kreek - Good call.  Implemented in https://github.com/department-of-veterans-affairs/vets-api/pull/1718/commits/b5f8f08f2d9d42dceba3e544bcd4c9c0cc71b08c